colossus/src/main/scala/colossus/controller/Pipe.scala

```diff
@@ -22,10 +22,27 @@ object Copier {
   }
 }
 
+//hrm....
+object ReflexiveCopiers{
+
+  trait Reflect[T] {
+    def copy(t : T) : T = t
+  }
+
+  implicit object StringReflect extends Copier[String] with Reflect[String]
```

Comment:
I'm pretty sure you can make a fully generic identity copier by having a type parameterized method and putting a default value on the implicit parameter that calls it.  I did that with `TagDecorator` and it seems to work well